apps/thunderhub/data/entrypoint.sh

```diff
@@ -3,4 +3,4 @@
 # Set password
 sed -i 's/$APP_PASSWORD/'${APP_PASSWORD}'/' /data/thubConfig.yaml
 
-exec npm start
+exec npm run start:prod
```

Comment:
@lukechilds Have you tried running the docker image in umbrel with this entrypoint modification?

I know you said you receive the error running the container straight without umbrel but it might be worth a shot. I can attempt to chime in once my RasPi delivered.